recipes/spiceypy/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "spiceypy" %}
+{% set version = "2.0.0" %}
+{% set sha256 = "33db27531f7e8e31ce06970231b3265fe043b608d1404c9f43b2af14508713f3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.zip
```

Comment:
Would change this to `{{ name }}-{{ version }}.zip` for clarity.

Reply:
Actually, pypi.io download urls only work for .tar.gz I think. Changing.